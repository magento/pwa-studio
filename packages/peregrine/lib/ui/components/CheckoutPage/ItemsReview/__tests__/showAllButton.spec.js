import React from 'react';

import createTestInstance from '@magento/peregrine/lib/util/createTestInstance';

import ShowAllButton from '../showAllButton';

test('Snapshot test', () => {
    const tree = createTestInstance(<ShowAllButton onFooterClick={() => {}} />);

    expect(tree.toJSON()).toMatchSnapshot();
});
