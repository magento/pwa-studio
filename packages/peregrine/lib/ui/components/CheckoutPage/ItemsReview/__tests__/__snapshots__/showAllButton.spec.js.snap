// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test 1`] = `
<button
  onClick={[Function]}
>
  <span>
    <span>
      <mock-FormattedMessage
        defaultMessage="SHOW ALL ITEMS"
        id="checkoutPage.showAllItems"
      />
    </span>
    <span>
      <svg
        fill="none"
        height={24}
        stroke="currentColor"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="2"
        viewBox="0 0 24 24"
        width={24}
        xmlns="http://www.w3.org/2000/svg"
      >
        <polyline
          points="6 9 12 15 18 9"
        />
      </svg>
    </span>
  </span>
</button>
`;
